components/dashboard/src/App.tsx

```diff
@@ -148,6 +151,7 @@ function App() {
                 <Route path="/notifications" exact component={Notifications} />
                 <Route path="/plans" exact component={Plans} />
                 <Route path="/teams" exact component={Teams} />
+                <Route path="/new-team" exact component={NewTeam} />
```

Comment:
**question**: Shall we use a more implicit path like `/teams/new` and also reserve this URL?

Reply:
I'm not sure, because `/teams` is the old Team Plans page, while new team URLs look more like `/:slug/members`.

Let's leave it as is for now, and maybe adjust the URL later. 😇

Reply:
Now tracked in https://github.com/gitpod-io/gitpod/issues/4471